pallets/parachain-staking/src/weights.rs

```diff
@@ -237,23 +238,23 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			.saturating_add(T::DbWeight::get().reads(9 as Weight))
 			.saturating_add(T::DbWeight::get().writes(7 as Weight))
 	}
-	// If this takes up too much block space, run again on code
-	// - #743 benchmarks post reward payout optimization were 3x lower per collator,
-	// 15x lower per nominator
-	fn active_on_initialize(x: u32, y: u32) -> Weight {
+	fn round_transition_on_initialize(x: u32, y: u32) -> Weight {
 		(0 as Weight)
-			// Standard Error: 299_000
-			.saturating_add((208_550_000 as Weight).saturating_mul(x as Weight))
-			// Standard Error: 27_000
-			.saturating_add((15_580_000 as Weight).saturating_mul(y as Weight))
-			.saturating_add(T::DbWeight::get().reads(26 as Weight))
-			.saturating_add(T::DbWeight::get().reads((4 as Weight).saturating_mul(x as Weight)))
-			.saturating_add(T::DbWeight::get().writes(16 as Weight))
-			.saturating_add(T::DbWeight::get().writes((4 as Weight).saturating_mul(x as Weight)))
-	}
-	fn passive_on_initialize() -> Weight {
+			// Standard Error: 1_342_000
+			.saturating_add((46_120_000 as Weight).saturating_mul(x as Weight))
+			// Standard Error: 12_000
+			.saturating_add((1_248_000 as Weight).saturating_mul(y as Weight))
+			.saturating_add(T::DbWeight::get().reads((2 as Weight).saturating_mul(x as Weight)))
+			.saturating_add(T::DbWeight::get().writes((2 as Weight).saturating_mul(x as Weight)))
+	}
+	fn base_on_initialize() -> Weight {
 		(4_913_000 as Weight).saturating_add(T::DbWeight::get().reads(1 as Weight))
 	}
+	fn pay_one_collator_reward(_y: u32) -> Weight {
+		(82_496_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(11 as Weight))
+			.saturating_add(T::DbWeight::get().writes(6 as Weight))
+	}
```

Comment:
it is odd that the cost of paying out one collator does not depend on the number of delegations to be paid out

Reply:
They were previously, I definitely need to revisit this (before this is merged). I'm also having trouble running the benchmarks on the benchmark server...